packages/app/src/providers/hsds/hsds-api.ts

```diff
@@ -161,12 +185,9 @@ export class HsdsApi extends ProviderApi {
       `/${entityCollection}/${entityId}/attributes`
     );
 
-    const attrsPromises = data.attributes.map(async (attr) => {
-      const { data } = await this.client.get<HsdsAttributeResponse>(
-        `/${entityCollection}/${entityId}/attributes/${attr.name}`
-      );
-      return data;
-    });
+    const attrsPromises = data.attributes.map(async (attr) =>
+      this.fetchAttributeWithValue(entityCollection, entityId, attr.name)
+    );
```

Comment:
The `fetchAttributes` method currently fetches all the attributes' metadata at once, and then each attribute's value one at a time like this. At the end of this refactoring, this look will be gone from here and only occur in `getAttrValues`.